src/main/java/dev/ebullient/convert/tools/dnd5e/Json2QuteFeat.java

```diff
@@ -42,4 +49,96 @@ enum FeatFields implements JsonNodeReader {
         category,
         ;
     }
+
+    public String getAbilityScoreIncreases(JsonNode abilityNode) {
```

Comment:
Likewise, I tend to keep the field enum last in the class file. Makes it easy to find. I group other methods .. sometimes in order of parsing, and sometimes in order of what they're doing. So this should move up closer to where other feat bonuses are parsed.